backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/RunVmCommandBase.java

```diff
@@ -83,7 +83,7 @@
     @Inject
     protected SchedulingManager schedulingManager;
     @Inject
-    private ResourceManager resourceManager;
+    protected ResourceManager resourceManager;
```

Comment:
how about introducing a `getVdsManager` method that similar to VmCommand::getVmManager instead?